fvm/evm/emulator/emulator.go

```diff
@@ -530,12 +545,14 @@ func (proc *procedure) deployAt(
 	var err error
 	inter := gethVM.NewEVMInterpreter(proc.evm)
 	contract := gethVM.NewContract(
-		gethVM.AccountRef(callerCommon),
-		gethVM.AccountRef(addr),
+		callerCommon,
+		addr,
 		castedValue,
-		call.GasLimit)
+		call.GasLimit,
+		nil,
```

Comment:
For the deployment of COAs, we pass `nil` for the `jumpDests` argument, as it is not exposed through the `EVM` object. See also:
- https://github.com/onflow/go-ethereum/blob/master/core/vm/contract.go#L49-L54
- https://github.com/onflow/go-ethereum/blob/master/core/vm/evm.go#L503-L505